test-schemas/src/main/resources/openapi/simple.yaml

```diff
@@ -100,3 +100,7 @@ components:
       properties:
         property1:
           type: string
+          required: true
+          x-obfuscated: true
```

Comment:
This file and related tests should not be used for testing dto-specific logic.
Testing should be done in io.github.fomin.oasgen.java.dto.jackson.wstatic.BuiltinConverterMatchersTest.

Reply:
dto-specific param is also processed, i revert this file and add test to ObjectConverterMatcherTest

Reply:
done; here we have dto-specific; property-specific test added to ObjectConverterMatcherTest